anvil-rstudio-bioconductor/scripts/set_up_package_dir.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -e
+
+# This script needs to be separated from the Dockerfile because we mount persistent disks in the
+# /home/rstudio directory and when we mount the PD, everything in that directory gets wiped.
+# Therefore, we will run this script from Leonardo after the PD is mounted
+
+RSTUDIO_USER_HOME=/home/rstudio
+R_PATH=`R CMD printenv R_LIBS_USER | sed 's/~\///g'`
```

Comment:
Could this be `printenv R_LIBS_USER | sed 's/~\///g'` ?

E.g. prob don't need to use R if just printing an ev

Reply:
based on my understanding, `R_LIBS_USER` is an environment variable set by R so that's why we used `R CMD` where as `BIOCONDUCTOR_DOCKER_VERSION` is not so we don't use `R CMD`

Reply:
This isn't a publicly available env variable, only accessible through R. 

For example,

```
$ docker run -it --rm us.gcr.io/anvil-gcr-public/anvil-rstudio-bioconductor:0.0.9 printenv
PATH=/usr/lib/rstudio-server/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
HOSTNAME=6a5df70fe63b
TERM=xterm
R_VERSION=4.0.3
LC_ALL=en_US.UTF-8
LANG=en_US.UTF-8
R_HOME=/usr/local/lib/R
CRAN=https://packagemanager.rstudio.com/all/__linux__/focal/latest
TZ=Etc/UTC
S6_VERSION=v1.21.7.0
RSTUDIO_VERSION=latest
DEBIAN_FRONTEND=noninteractive
BIOCONDUCTOR_DOCKER_VERSION=3.12.30
RSTUDIO_PORT=8001
RSTUDIO_HOME=/etc/rstudio
TERRA_R_PLATFORM=anvil-rstudio-bioconductor
TERRA_R_PLATFORM_BINARY_VERSION=0.99.1
RSTUDIO_USERSETTING=/home/rstudio/.rstudio/monitored/user-settings/user-settings
HOME=/root
```

Reply:
oh ok 👍